src/app/drive/components/DriveExplorer/DriveExplorerItem/DriveExplorerGridItem/DriveExplorerGridItem.scss

```diff
@@ -25,6 +25,11 @@
         height: 50%;
         width: 50%;
       }
+
+      .file-thumbnail {
+        height: 100%;
```

Comment:
Maybe with this we can use tailwind's `w-full h-full` so we keep everything in the same file